src/pages/RequestCallPage.js

```diff
@@ -102,19 +99,23 @@ class RequestCallPage extends Component {
             return;
         }
 
-        const personalPolicy = _.find(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.PERSONAL);
-        if (!personalPolicy) {
+        // If there's a lastAccessedWorkspace in Onyx then use that as the policy for the call, otherwise use the personal policy
+        const policyForCall = this.props.lastAccessedWorkspace
+            ? this.props.lastAccessedWorkspace : _.find(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.PERSONAL);
```

Comment:
Just a thought, but what if we change this to guarantee a `policy` is returned like so? 

```js
const policyForCall =_.find(this.props.policies, policy => {
    if (!policy) {
        return;
    }
    
    if (this.props.lastAccessedWorkspacePolicyID) {
        return policy.id === this.props.lastAccessedWorkspacePolicyID;
    }

    return policy.type === CONST.POLICY.TYPE.PERSONAL;
});
```

Then we won't need line `111` and also the `policyForCall` variable will no longer ambiguously refer to either a "policy" or "policyID" type?

Reply:
smart, thanks!